### PR TITLE
Selenium: remove info about known 10177 and 10699 issues

### PR DESCRIPTION
**Pull request by SkorikSergey**
### What does this PR do?
This PR: 
- remove info about known #10177 and #10699 issues;
- move checking known #10674 issue from **CodenvyEditor** page object to tests.

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/11732

### CODE REVIEW

**Comment by SkorikSergey** [comment]
ci-test

**Comment by riuvshin** [comment]
Results of automated E2E tests of Eclipse Che Multiuser on OCP:
[Build details](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/1075/)
[Test report](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/1075//Selenium_20tests_20report/)
docker image: eclipseche/che-server:11734
https://github.com/orgs/eclipse/teams/eclipse-che-qa please check this report.

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java [hunk 1]
```diff
@@ -2368,4 +2355,36 @@ public void doRenamingByLanguageServerField(String renameValue) {
...
+   * @param expectedProposal text which should be present in the container
+   */
+  public void waitProposalIntoSignaturesContainer(final String expectedProposal) {
+    webDriverWaitFactory
+        .get(ELEMENT_TIMEOUT_SEC)
+        .until(
+            (ExpectedCondition<Boolean>)
+                webDriver -> getAllVisibleTextFromSignaturesContainer().contains(expectedProposal));
+  }
+  ///
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java]: looks redundant here

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java [hunk 2]
```diff
@@ -2368,4 +2355,36 @@ public void doRenamingByLanguageServerField(String renameValue) {
...
+    return signaturesContainer.getText();
+  }
+
+  /**
+   * Waits specified {@code expectedProposal} in signatures container.
+   *
+   * @param expectedProposal text which should be present in the container
+   */
+  public void waitProposalIntoSignaturesContainer(final String expectedProposal) {
+    webDriverWaitFactory
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java]: Let's use seleniumWebDriverHelper here

**Comment by riuvshin** [comment]
Results of automated E2E tests of Eclipse Che Multiuser on OCP:
[Build details](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/1088/)
[Test report](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/1088//Selenium_20tests_20report/)
docker image: eclipseche/che-server:11734
https://github.com/orgs/eclipse/teams/eclipse-che-qa please check this report.

**Review by dmytro-ndp — APPROVED** [review]

**Review by artaleks9 — APPROVED** [review]

**Pull request merged**